core/src/execution/mod.rs

```diff
@@ -197,6 +197,7 @@ impl<N: NetworkSpec, R: ExecutionRpc<N>> ExecutionClient<N, R> {
 
         let tx_hashes = block.transactions.hashes();
 
+        // TODO: replace this with rpc.get_block_receipts?
```

Comment:
@ncitron; thoughts on this?

Using `getBlockReceipts` instead of `getTransactionReceipt` would significantly reduce the number of HTTP calls made to the RPC.